docs/intro.md

```diff
@@ -0,0 +1,52 @@
+
+- **Privacy**
+- **Cost & Time Savings**
+- **Scaling**
+- **Flexibility & Extensibility**
+
+In a typical on-chain bitcoin transaction, all of the [UTXOs](https://unchained.com/blog/what-is-a-utxo-bitcoin/) would belong to Alice. After all, one might ask, what sense does it make for Bob to send bitcoin to himself? It is _assumed_ that the spender in a transaction is the one who owns every UTXO:
+
+![Assumed Transaction](./img/regular.png)
+
+This assumption is called the [common-input-ownership heuristic](https://en.bitcoin.it/wiki/Common-input-ownership_heuristic), and even Satoshi declared it the last unsolved privacy problem in the [whitepaper](https://bitcoin.org/bitcoin.pdf). But it wasn't unsolved, just undiscovered. Because the common-input-ownership heuristic is not required, a transaction can be constructed with any number of participants spending to anyone, including to themselves. One such combination is a [CoinJoin](https://en.bitcoin.it/wiki/CoinJoin), whose primary purpose is for the participants to collaboratively construct transactions to themselves to gain privacy. But it comes at the cost of paying high fees and requires high user interaction.
+
+![CoinJoin](./img/coinjoin.png)
+
+Payjoin is a different combination, which uses both sender and receiver UTXOs:
+
+![Payjoin](./img/Payjoin.png)
+
+### Privacy
+
+Payjoin also breaks the common-input-ownership heuristic, which is the primary metric used by creepy blockchain analysis companies trying to spy on your financial life. In this way, it restores and re-normalizes privacy as a fundamental human right back to the common individual, breaking the mold of difficult to use, expensive, centralized CoinJoin coordinators which can invite bad actors <!-- include this last part??? -->. With Payjoin, privacy best practices can be seamlessly integrated into the wallet experience, it is not a requirement that the user be aware they are conducting a Payjoin. Privacy is easy if it's obtained by _default_.
+
+### Cost & Time Savings
+
+Payjoin can save money and time by batching transactions when fees are low. Most wallets default to [address reuse](https://en.bitcoin.it/wiki/Address_reuse) (as they should!). This however results in more UTXOs when spending a transaction. Since the scarce resource in Bitcoin is block space, the [fee](https://unchained.com/blog/bitcoin-transaction-fees/) paid (in satoshis) in a transaction is measured relative to the number of bytes in the transaction, i.e. satoshis per byte or [satoshis per virtual byte](https://bitcoin.stackexchange.com/questions/89385/is-there-a-difference-between-bytes-and-virtual-bytes-vbytes). So if you have many small UTXOs, you're likely to have to combine some to make larger payments, which results in higher fees. Therefore, you should periodically [consolidate UTXOs](https://unchained.com/blog/too-many-bitcoin-utxos/) by sending them to a new address you own when fees are low.
```

Comment:
why should most wallets default to address reuse? And why does that result in more UTXOs (and more than what?)

passive voice "the fee ... is measured" could be active

Reply:
>  why should most wallets default to address reuse? And why does that result in more UTXOs (and more than what?)
Jeez, I meant the exact opposite of what I actually wrote here. My mistake for not catching this obvious mistake on reread. What I meant was:

> Most wallets default to generating new addresses for every payment to avoid address reuse (as they should!)

Will fix